enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,144 @@
+---
+title: monitoring-windows-nodes
+authors:
+  - "@VaishnaviHire"
+  - "@ravisantoshgudimetla"
+reviewers:
+  - "@@openshift/openshift-team-windows-containers"
+approvers:
+  - "@sdodson"
```

Comment:
Please add Simon from the monitoring team

Reply:
Should I add him as a `reviewer` or `approver` ?

Reply:
Both. We also need to add someone from the console team as a reviewer given they are tangentially involved.